incubator/hnc/internal/validators/hncconfig_test.go

```diff
@@ -48,7 +51,8 @@ func TestDeletingOtherObject(t *testing.T) {
 }
 
 func TestRBACTypes(t *testing.T) {
-	config := &HNCConfig{}
+	f := forest.NewForest()
+	config := &HNCConfig{Forest: f}
```

Comment:
Delete?

Reply:
It will have nil pointer error if we have forest as empty.